### ISSUE TITLE
ubuntu 22.04.1LTS pdfstitcher v0.8.3 Settings->Preferences error

### ISSUE THREAD

**Issue by bluedarter2** [cfcurtis/pdfstitcher#182]
pdfstitcher v0.8.3 is installed via flatpak.  I ran into this error using v0.8.1 (traceback line#s differed, but that's to be expected).  I had flatpak update to v0.8.3.

This of course is not a showstopper, I just thought I'd report it before I forgot about it.

Thanks so much for writing and maintaining this Charlotte!
Regards,
Evelyn

## Steps to reproduce the problem

1. Open pdfstitcher
2. Select Settings->Preferences

## Program Output
```
Traceback (most recent call last):
  File "/app/lib/python3.10/site-packages/pdfstitcher/ui/main_frame.py", line 286, in on_prefs
    prefs_dia = PrefsDialog(parent=self)
  File "/app/lib/python3.10/site-packages/pdfstitcher/ui/dialogs.py", line 301, in __init__
    self.lang_combo.SetSelection(utils.valid_langs.index(Config.general["language"]))
ValueError: None is not in list
```

## Current Configuration
```
general:
  check_updates: true
  language: null
  margin: '0'
  open_dir: ''
  save_dir: ''
  units: !units 'INCHES'
line_props:
  colour:
    enable: true
    fill: false
    value: !!python/tuple
    - 0
    - 0
    - 0
  style:
    enable: true
    value: 0
  thickness:
    enable: true
    units: !units 'POINTS'
    value: '4'
```

## System Info
```
architecture: x86_64
pdfstitcher-version: 0.8.3
platform: Linux
platform-release: 5.19.0-35-generic
platform-version: '#36~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Fri Feb 17 15:17:25 UTC
  2'
processor: ''
python-build: !!python/tuple
- main
- Nov 10 2011 15:00:00
python-implementation: CPython
python-version: 3.10.6
ram: 31 GB
```

**Comment by cfcurtis** [comment]
Thanks for the report! It seems the system language is not being detected with the Flatpak version, I'll investigate that and at least make it default to English like the rest of the UI.

**Comment by cfcurtis** [comment]
Fixed in 0.8.4
